tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -2313,7 +2295,6 @@ Status StorageManager::load_consolidated_fragment_meta(
   auto meta_name = uri.remove_trailing_slash().last_path_part();
   auto pos = meta_name.find_last_of('.');
   meta_name = (pos == std::string::npos) ? meta_name : meta_name.substr(0, pos);
-  RETURN_NOT_OK(utils::parse::get_fragment_version(meta_name, meta_version));
```

Comment:
The 4 lines above seem redundant as well.

Reply:
Removed